requirements/build.txt

```diff
@@ -1,2 +1,3 @@
 Cython>=0.23
 wheel
+versioneer
```

Comment:
It appears you are vendoring versioneer while also adding it as a dependency?

Reply:
hmm, I think you are right. We don't need it as a dependency. Good catch.